src/backend/core/models/workflow_metrics_models.py

```
@@ -262,11 +230,10 @@ def track_keyword_search(self, latency_ms: float, returned_keywords: List[str])
         )
         self.keyword_search_stats["returned_keywords"].extend(returned_keywords)
     
-    def track_pattern_learning(self, predicted: bool, keyword_count: int, accuracy: float = 0.0) -> None:
+    def track_pattern_learning(self, predicted: bool, keyword_count: int) -> None:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)